packages/vscode/src/extension.ts

```diff
@@ -28,7 +28,6 @@ import {
   isDebugOrTestSession,
   enablePrismaNodeModulesFolderWatch,
 } from './util'
-import * as d from 'lsp/cli.js'
```

Comment:
This was what was causing the failure, so instead of `import`ing it, I'll require it down below